src/controllers/user.controller.js

```diff
@@ -0,0 +1,96 @@
+import { ApiError } from '../exeptions/api.error.js';
+import { User } from '../modules/User.js';
+import { userService } from '../service/user.service.js';
+import bcrypt from 'bcrypt';
+
+const getAllActivated = async (req, res) => {
+  const users = await userService.getAllActivated();
+
+  res.send(users.map(userService.normalize));
+};
+
+const changeName = async (req, res) => {
+  const { userName } = req.params;
+  const { newName } = req.body;
+
+  if (!userName || !newName) {
+    throw ApiError.badRequest('Something wrong');
+  }
+
+  await userService.changeName(userName, newName);
+  res.send('Name change sucssessfull');
```

Comment:
There is a typo in 'Name change sucssessfull'. It should be 'Name change successful'.